### ISSUE TITLE
Sequence Detail Page: Concordances table lacking `class="table-responsive"` div

### ISSUE THREAD

**Issue by jacobdgm** [DDMAL/CantusDB#1355]
![Screenshot 2024-03-12 at 10 31 57 AM](https://github.com/DDMAL/CantusDB/assets/58090591/9929fcef-c61d-4591-8bd4-aea3487e5b54)
